cli/cli_test.go

```diff
@@ -157,3 +157,32 @@ func TestModuleImageWithoutPayload(t *testing.T) {
 	assert.NoError(t, err)
 	assert.Empty(t, string(outputBytes))
 }
+
+func TestWriteBootstrapArtifact(t *testing.T) {
+	app := getCliContext()
+
+	menderFile, err := ioutil.TempFile("", "")
+	require.NoError(t, err)
+	menderFile.Close()
+	menderName := menderFile.Name()
+	defer os.Remove(menderName)
+
+	// Default
+	err = app.Run([]string{"mender-artifact",
+		"write",
+		"bootstrap-artifact",
+		"-t", "dummy",
+		"-n", "dummy",
+		"-G", "dep_gr",
+		"-g", "pr_gr",
+		"--clears-provides", "cl_pr",
+		"-d", "dep:val",
+		"-p", "pr:val",
+		"-o", menderName,
+	})
```

Comment:
AFAICS, we have nothing which tests the reading part. This is evident in that the [reader code is never hit](https://coveralls.io/jobs/102652485/source_files/23088432295#L687). We should use the `read` command at least once to verify that it works correctly for empty payloads.

OTOH the high level `read` command could be a bit clunky to use when checking the details, so I suppose there should be a few tests which calls the reader directly and verifies that the returned data is correct. I'll leave the details to you to decide what's most convenient.

Reply:
Done! Please take a look and see if current solution satisfies you.

Reply:
👍